Frontend/src/components/TableEditors/TableToCSVExporter.svelte

```diff
@@ -0,0 +1,133 @@
+<script>
+  import Database from "../Database/ENV_DATABASE";
+  import { location } from "svelte-spa-router";
+  import itemColumns from "./Items/Columns";
+  import customerColumns from "./Customers/Columns";
+  import rentalColumns from "./Rentals/Columns";
+
+  const columns = {
+    rental: rentalColumns,
+    item: itemColumns,
+    customer: customerColumns,
+  };
+
+  const filenames = {
+    rental: "Leihvorgaenge",
+    item: "Gegenstaende",
+    customer: "Kunden",
+  };
+
+  // rental, item or customer
+  $: itemType = $location.split("/")[1].slice(0, -1);
```

Comment:
this seems a bit hacky, but I assume there is no other way to get the type?

Reply:
We could also store the type in a svelte store, but then we would have to update this store manually, that‘s why I chose this way. 

If we keep the current url style, e.g. `/items` and just append new stuff like `/items/123` or `/items?id=123` it will not break. It would only break if we change the `/items` part to something else.